contactaccessrights/js/src/access-rights.js

```diff
@@ -0,0 +1,22 @@
+(function () {
+  var extPath = CRM.vars.contactAccessRights.baseURL + '/js/src/access-rights';
+  require.config({
+    urlArgs: 'bust=' + (new Date()).getTime(),
+    paths: {
+      'access-rights': extPath
+    }
+  });
+
+  require(['access-rights/app'], function (angular) {
+    var actions = angular.element('.crm-actions-ribbon ul#actions');
```

Comment:
Please encapsulate all this DOM manipulation in a function (or even better, a separate RequireJS module) where you document it and explain what and why you do this